### DISCUSSION TITLE
Severity 10 due to end of life Scan Engine

### DISCUSSION THREAD

**Discussion by facbr** [Secure-Compliance-Solutions-LLC/GVM-Docker#165] [Feature Request]
Hello
Since 2 days, my GVM report an end of life scan engine as decribed below

Detection Result
Installed GVM Libraries (gvm-libs) version:        20.8.0
Latest available GVM Libraries (gvm-libs) version: 20.8.1
Reference URL(s) for the latest available version: https://community.greenbone.net/t/gvm-20-08-stable-initial-release-2020-08-12/6312

This message is included in all scans and all reports are now in red with severity 10 :(
I am using the last docker version posted in docker hub - 2021-02-25 22:02:42 but still in version 20.8.0
How can I upgrade to v 20.8.1 or later ?
Do you plan to deliver a new version soon ?
Thanks for your help and your great job
Regards

**Comment by pixelsquared** [comment]
We just released a new version to fix this try to update the container

**Reply by facbr** [reply]
It's working now. Many Thanks for your help and reactivity
Best regards
